src/components/ChatEntry.js

```diff
@@ -1,22 +1,38 @@
 import React from 'react';
 import './ChatEntry.css';
 import PropTypes from 'prop-types';
+import TimeStamp from './TimeStamp';
 
-const ChatEntry = (props) => {
+
+const ChatEntry = ({id, sender, body, timeStamp, liked, updateIsLiked}) => {
+  const entryClass = (id % 2 === 0) ? 'remote':'local';
+  const hearts = liked ? '❤️': '🤍' ;
+  
   return (
-    <div className="chat-entry local">
-      <h2 className="entry-name">Replace with name of sender</h2>
+    <div className= { 'chat-entry' + ' ' + entryClass}>
```

Comment:
```suggestion
    <div className= { 'chat-entry ' + entryClass}>
```
you can add the space after `chat-entry` to avoid doing two concatenations